itou/scripts/tests.py

```diff
@@ -1,12 +1,16 @@
 import datetime
 import io
+import tempfile
```

Comment:
Peut-être plutôt utiliser la fixture [`tmp_path` de pytest](https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html) plutôt que de créer des fichiers dans le dossier temporaire de la machine locale, histoire de s'éviter des problèmes (`xdist` :eyes:).
Surtout que maintenant il y a un truc qui facilite l'utilisation des fixtures pytest pour les `TestCase` :
https://github.com/betagouv/itou/blob/ad184ed43f88c439efd87ff7732e3d4fd76f616d/itou/conftest.py#L91-L93

Reply:
Top, merci ! J'en ai profité pour utiliser Pytest (j'avais oublié... :grimacing: ).